### ISSUE TITLE
DefaultValue is set to '' when the value is false (for custom editor)

### ISSUE THREAD

**Issue by AnishJoseph** [AllenFang/react-bootstrap-table#1254]
I have a custom editor - the values it accepts are boolean - true/false. 
In TableEditColumn.js, line #176, there is this line 
```javasctipt
        defaultValue: fieldValue || '',
```
When the fieldValue happens to be false, defaultValue will get set to ''. 
This line should probably be
```javascript
defaultValue : fieldValue === undefined ? '' : fieldValue
```

**Comment by AllenFang** [comment]
ok, I'll handle it on next version, thanks

**Comment by AnishJoseph** [comment]
Wonderful - thanks

**Comment by AllenFang** [comment]
Fixed on `v3.3.0`, thanks :)
